backend/LexBoxApi/Auth/AuthKernel.cs

```diff
@@ -136,6 +136,23 @@ public static class AuthKernel
                         return Task.CompletedTask;
                     }
                 };
+            }).AddGoogle(googleOptions =>
+            {
+                var googleConfig = configuration.GetSection("Authentication:Google").Get<GoogleOptions>();
+                // ArgumentNullException.ThrowIfNull(googleOptions); // Eventually we'll throw if google config not found
+                if (googleConfig is not null)
+                {
+                    googleOptions.ClientId = googleConfig.ClientId;
+                    googleOptions.ClientSecret = googleConfig.ClientSecret;
+                }
+                googleOptions.CallbackPath = "/api/login/signin-google";
+                googleOptions.Events.OnTicketReceived = async context =>
+                {
+                    context.HandleResponse();
+                    var lexAuthService = context.HttpContext.RequestServices.GetRequiredService<LexAuthService>();
+                    var redirectTo = await Controllers.LoginController.CompleteGoogleLogin(context.Principal, context.Properties.RedirectUri, context.HttpContext, lexAuthService);
```

Comment:
It would probably be easier to get the LoginController from the IoC container, then you can reduce the parameters here and make it an instance method. Then all you need to pass in is the principal and redirect url.

Reply:
I tried, and the IoC container said "No such service". That's why I ended up making it a static method, because I couldn't figure out where the controllers get registered. It works, but yes, you're right that it would be better to just get the LoginController from the container. If I could, I would.

Reply:
Fixed in commit 0d9e1c2f. Thanks for pointing me to the `.AddControllersAsServices` method, which is what I was missing.